### ISSUE TITLE
> i have the same issue, did you found a solution?

### ISSUE THREAD

**Issue by kazak01xd** [wifiphisher/wifiphisher#1389]
> i have the same issue, did you found a solution?
Still waiting for the advice
If you get some solutions, please share it here.

_Originally posted by @secuhiro in https://github.com/wifiphisher/wifiphisher/issues/1388#issuecomment-720049099_

have you tried using pip3 to install setuptools?
so open up a new terminal and type this in:

sudo apt-get install python3-pip -y
sudo apt-get update
pip3 install setuptools

try that and see if that installs the needed utilities and try to continue the wifiphisher setup as normal. if that doesn't work let me know and I will try and help you work out the problem as best as I can.

**Comment by secuhiro** [comment]
Thank you for your advaice!!

"sudo python3 setup.py install" works fine

**Comment by kazak01xd** [comment]
> Thank you for your advaice!!
> 
> "sudo python3 setup.py install" works fine

awesome! were you able to set up your wifiphisher? is everything work as it should now?

**Comment by secuhiro** [comment]
I really appreciate your cooperation. Thank you very much!!
Set up is OK.
So I run "sudo wifiphisher" after changing Monitor mode
But nothing has picked up.
Please see in detail at attached screen shot.
![2020-11-01_19h20_00](https://user-images.githubusercontent.com/73603024/97800363-03c7a180-1c78-11eb-880b-ab7c3c1e24d1.png)
![2020-11-01_19h19_27](https://user-images.githubusercontent.com/73603024/97800367-0629fb80-1c78-11eb-8cfc-0aa8c697e344.png)

**Comment by kazak01xd** [comment]
> I really appreciate your cooperation. Thank you very much!!
> Set up is OK.
> So I run "sudo wifiphisher" after changing Monitor mode
> But nothing has picked up.
> Please see in detail at attached screen shot.
> ![2020-11-01_19h20_00](https://user-images.githubusercontent.com/73603024/97800363-03c7a180-1c78-11eb-880b-ab7c3c1e24d1.png)
> ![2020-11-01_19h19_27](https://user-images.githubusercontent.com/73603024/97800367-0629fb80-1c78-11eb-8cfc-0aa8c697e344.png)

interesting. if could be a hardware compatibility issue; I see there are other issues but maybe check and see if monitor mode works well with legacy tools like airdump-ng and see if your able to capture traffic with your chipset set to monitor mode. it's always a safe bet before going on a wild goose chase when it turns out its a simple hardware or even driver issue. happened to me trying to plug a usb 3.0 wifi adapter into a usb 2.0 and I was having power issues and like a dummy I kept blaming the driver and not my hardware. check the output of your iw inside a terminal window as well and tell me your chipset not the brand name and as well use airdump-ng to capture packets and if that doesn't work it could be a hardware problem if it does then we've found another bug that needs to be fixed on wifiphisher.

**Comment by secuhiro** [comment]
Thank you for your support!!

airdump-ng works fine and detail info within attached file.
Please take a look if there are any problems
![2020-11-02_03h21_04](https://user-images.githubusercontent.com/73603024/97811370-8f175600-1cbd-11eb-8531-eeeb5af64cd9.png)
![2020-11-02_03h30_05](https://user-images.githubusercontent.com/73603024/97811372-9179b000-1cbd-11eb-83d7-e6f62bdb7dd7.png)
![2020-11-02_03h39_48](https://user-images.githubusercontent.com/73603024/97811374-93437380-1cbd-11eb-9d5a-e5aec3dfe32d.png)

**Comment by kazak01xd** [comment]
Yeah so it does seem to be a problem with wifiphisher then. ok good. let me see if I can reproduce the same bug and see if I can find were the problem is happening. will report back asap

**Comment by kazak01xd** [comment]
> Thank you for your support!!
> 
> airdump-ng works fine and detail info within attached file.
> Please take a look if there are any problems
> ![2020-11-02_03h21_04](https://user-images.githubusercontent.com/73603024/97811370-8f175600-1cbd-11eb-8531-eeeb5af64cd9.png)
> ![2020-11-02_03h30_05](https://user-images.githubusercontent.com/73603024/97811372-9179b000-1cbd-11eb-83d7-e6f62bdb7dd7.png)
> ![2020-11-02_03h39_48](https://user-images.githubusercontent.com/73603024/97811374-93437380-1cbd-11eb-9d5a-e5aec3dfe32d.png)

one thing you could try in the mean time is to actually try and download it not from the github repo but actually using sudo apt-get to install it directly from kali repo.
so if you do a "sudo apt-get install wifiphisher -y" and install it from there put your wifi card into monitor mode and then use that. also are you using kali on a virtual box? because the only time I've ever gotten any wifi hacking tools to work is when I live booted or installed kali on my laptop and then ran my wifi card (external) with drivers it came with or I downloaded and not the vbox drivers since they are notoriously buggy. try this in the mean time will I try and find the issue; download it with apt-get and install it (its possible its a later version without this bug) and then put your card into monitor mode and type wifiphisher into the terminal and try and use it like that. if the same issue arises we could have a bigger bug and not some configuration issue.

**Comment by secuhiro** [comment]
Awesome support!!

I tried  
sudo apt-get install wifiphisher -y
then 
sudo wifiphisher

It was same result( nothing has picked up)
But just in case, I tried once more,
then picked up so many APs!!

I tried once more again,
nothing has picked up

After that, I tried once more,
then picked up so many APs

Next, nothing has picked up
Next, picked up so many APs 

It seems that
sudo wifiphisher does work and not work in turns

Anyway Now I am able to use  wifiphisher. I am really happy with that.
Thank you very very much!!

**Comment by kazak01xd** [comment]
> Awesome support!!
> 
> I tried
> sudo apt-get install wifiphisher -y
> then
> sudo wifiphisher
> 
> It was same result( nothing has picked up)
> But just in case, I tried once more,
> then picked up so many APs!!
> 
> I tried once more again,
> nothing has picked up
> 
> After that, I tried once more,
> then picked up so many APs
> 
> Next, nothing has picked up
> Next, picked up so many APs
> 
> It seems that
> sudo wifiphisher does work and not work in turns
> 
> Anyway Now I am able to use wifiphisher. I am really happy with that.
> Thank you very very much!!

Awesome man! Yeah that's super weird I'll be looking at this issue further. sadly for me I lent my alfa card to a friend that doesn't have one and now I need to buy another one for myself in the mean time. I'll be looking at this tools closely and I'll keep trying to find why it's doing whats it's doing. you should try for the sake of experiment to launch a test attack and see the results. I'm also seeing people posting about them able to see clients, launch the needed deauth but when they try and connect like trying to act as the victim; there is no traffic shown when they reach the fake AP password capture page. so try that and let me know how it works if you have the time. I'm curious as to why the main developers decided to almost abandon this project. hands down the best tool i've ever used for hacking wifi network or hacking in general. wish the rapid7 guys would add something like this to the metasploit framework.

**Comment by secuhiro** [comment]
>you should try for the sake of experiment to launch a test attack and see the results. 
If there's anything I can do for you, please let me know, I will try as much as I can.
Keep in touch!!
